src/main/java/duckling/responders/FolderContents.java

```diff
@@ -40,18 +40,23 @@ public boolean matches() {
     public ArrayList<String> headers() throws IOException {
         ResponseHeaders headers = new ResponseHeaders();
 
-        if (methodNotAllowed()) return headers.notAllowed().toList();
+        if (!isAllowed()) return headers.notAllowed().toList();
 
         return headers.withContentType("text/html").toList();
     }
 
     @Override
     public InputStream body() throws IOException {
-        String responseContent = methodNotAllowed() ? "" : rawFolderResponse();
+        String responseContent = !isAllowed() ? "" : rawFolderResponse();
 
         return new ByteArrayInputStream(responseContent.getBytes());
     }
 
+    @Override
+    public boolean isAllowed() {
+        return this.allowedMethods.contains(request.getMethod());
+    }
+
```

Comment:
Yeah!  That'd be great.  I didn't implement it in a single spot because `Responder` is an abstract class.  Is there a different way I can define `Responder` which would let me do that?  Maybe move to inheritance even?